src/lib/getOrCreateAppFolderWithReference.spec.js

```diff
@@ -0,0 +1,33 @@
+import { models } from 'cozy-client'
+const { ensureMagicFolder, getReferencedFolder } = models.folder
+
+import { getOrCreateAppFolderWithReference } from 'src/lib/getOrCreateAppFolderWithReference'
+
+jest.mock('cozy-client', () => ({
+  ...jest.requireActual('cozy-client'),
+  models: {
+    folder: {
+      ensureMagicFolder: jest.fn(),
+      getReferencedFolder: jest.fn()
+    }
+  }
+}))
+
+const setup = referencedFilesRes => {
+  ensureMagicFolder.mockReturnValue(referencedFilesRes)
+  getReferencedFolder.mockReturnValue(referencedFilesRes)
```

Comment:
could you use different variable 🙏 ?

This would prevent a developer to change the code into
```
export const getOrCreateAppFolderWithReference = client => getReferencedFolder(client, {
    _id: APP_DIR_REF,
    _type: APPS_DOCTYPE
  })
```
because the current test won't fail